### ISSUE TITLE
a11y: Fix focus for generic base components

### ISSUE THREAD

**Issue by tshfx** [MetaMask/metamask-extension#6861]
**Describe the bug**
input and textarea have a forced `outline: 0` in the styles, this greatly impacts users relying on screen readers and other accessible devices. This should switch to either having focus only while tabbing or have a design that supports input focus's.

**To Reproduce**
Steps to reproduce the behaviour:
1. Go to Details
2. Click on Export Private Key
3. Tab through to the input field
4. The key cursor flashed but there is no focus indicator.

**Expected behavior**
There should be a focus indicator when tabbing through elements.

**Screenshots**
![Screen Shot 2019-07-15 at 9 32 44 PM](https://user-images.githubusercontent.com/52763493/61257595-20209180-a748-11e9-8abe-ac391c5706d5.png)

**Browser details (please complete the following information):**
 - OS: OS X
 - Browser: chrome
 - MetaMask Version: 6.7.2

**Additional context**
A11y refers to accessibility. This change would improve the extensions screen reader support.

**Comment by tmashuang** [comment]
Closing since #6871 has been merged.
